logic/src/commonTest/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapperProvider.kt

```diff
@@ -1,3 +1,2 @@
 package com.wire.kalium.logic.data.message
 
-expect fun provideProtoContentMapper(): ProtoContentMapper
```

Comment:
Why do we still have this class/file if it is not being used anymore? Can't we delete it? 🤔

Reply:
Oops hahahah

I'll delete it tomorrow when I'm back.